### DISCUSSION TITLE
Nominating @robdimsdale as a Builders ~~contributor~~ maintainer

### DISCUSSION THREAD

**Discussion by fg-j** [paketo-buildpacks/feedback#89] [Builder Team]
@robdimsdale is already a Paketo Stacks maintainer. It's a natural fit for him to begin taking on the next part of the Paketo image-distribution pipeline.

**Comment by robdimsdale** [comment]
100% agree. Thanks for nominating me!

**Comment by ryanmoran** [comment]
I assume this was meant to be a nomination for a maintainer role, since @robdimsdale is already a contributor, right?

**Comment by robdimsdale** [comment]
I think so!

**Comment by ForestEckhardt** [comment]
I would approve a maintainership as well

**Comment by robdimsdale** [comment]
I edited the title to reflect my desire to be a maintainer

**Comment by ryanmoran** [comment]
Ok. I'll also assume that we won't be getting votes from the other Builders maintainers and suggest that there should probably be a vote to have them moved to emeritus status.

**Comment by robdimsdale** [comment]
I can start that vote now that I'm a maintainer
